src/WCSimPrimaryGeneratorMessenger.cc

```diff
@@ -405,6 +439,7 @@ void WCSimPrimaryGeneratorMessenger::SetNewValue(G4UIcommand * command,G4String
     else if ( newValue == "gamma-conversion")
     {
       myAction->SetMulineEvtGenerator(false);
+      myAction->SetAmBeEvtGenerator(false);
```

Comment:
also need this line for `else if (newValue == "mPMT-LED")` below

Reply:
👍